tests/unit/meta/test_snap_yaml.py

```diff
@@ -925,3 +900,37 @@ def test_app_passthrough_snap_yaml(simple_project, new_dir):
           PATH: $SNAP/usr/sbin:$SNAP/usr/bin:$SNAP/sbin:$SNAP/bin:$PATH
         """
     )
+
+
+def test_architectures(simple_project, new_dir):
```

Comment:
Please test this with another arch triplet

Reply:
Done!